attribute-masking-service/src/main/java/uk/gov/gchq/palisade/service/attributemask/stream/PropertiesConfigurer.java

```diff
@@ -57,11 +57,11 @@
 public class PropertiesConfigurer extends PropertySourcesPlaceholderConfigurer implements InitializingBean {
 
     private static final Pattern INDEXED_PROPERTY_PATTERN = Pattern.compile("^\\s*(?<path>\\w+(?:\\.\\w+)*)\\[(?<index>\\d+)\\]\\.*(.*?)$");
```

Comment:
```suggestion
    private static final Pattern INDEXED_PROPERTY_PATTERN = Pattern.compile("^\\s*(?<path>\\w+(?:\\.\\w+)*)\\[(?<index>\\d+)\\]\\.*(?<tail>.*?)$");
```
Missing the tag for the third index, the TAIL

Reply:
Oops, added for all instances of this class.